idl/github.com/uber/cadence/shared.thrift

```diff
@@ -177,6 +177,10 @@ struct TaskList {
   10: optional string name
 }
 
+struct TaskListMetadata {
+  10: optional i64 maxTasksPerSecond
```

Comment:
It should be non integer. For example it should be possible to specify 0.1 per second to get one in 10 seconds.